packages/dev-tools/src/actions/WriteSummary.tsx

```diff
@@ -56,6 +56,17 @@ export function WriteSummary({ write }: Props) {
           .filter(isDefined)
       : null;
 
+  let functionName = write.request.functionName;
+  let functionArgs = write.request.args;
+  if (functionName === "call" || functionName === "callFrom") {
```

Comment:
note that `call` and `callFrom` have different signatures:
https://github.com/latticexyz/mud/blob/4c7fd3eb29e3d3954f2f1f36ace474a436082651/packages/world/src/interfaces/IWorldKernel.sol#L16-L32

Reply:
I see. Looks like funcSelectorAndArgs is the last param for both so should be ok here?

Reply:
oh yep, you're right, I missed that you were grabbing the last element, not slicing off the first element 🙈 my bad